### DISCUSSION TITLE
install in conda

### DISCUSSION THREAD

**Discussion by dinosg** [blue-marble/gridpath#1056] [Q&A] [answered]
the documentation says to use pip to install- but that's generally a bad practice in anaconda, circumventing version control and compatibility in conda... is there a 'conda install' type installation process for this? Thx...

**Comment by anamileva** [comment, marked as answer]
We develop and test on the CPython implementation of Python only.

Using the Anaconda distribution instead may or may not work. At a minimum, you'd need to `conda install pip` first. https://stackoverflow.com/a/43729857
